### PR TITLE
add msa support

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
 # Release History
 
-## 1.14.1 (Unreleased)
+## 1.15.0b1 (Unreleased)
 
 ### Features Added
 
+- Added Windows Web Account Manager (WAM) Brokered Authentication support.
+- Added `enable_msa_passthrough` suppport for `InteractiveBrowserCredential`.
```
Comment (joshfree) [inline sdk/identity/azure-identity/CHANGELOG.md:8]: Could expand this feature description, similar to this comment: https://github.com/Azure/azure-cli/pull/24181

**Review by joshfree — APPROVED** [review]
Could you also add an entry to the TSG for MSA passthrough.  You can reuse the account screenshots from here: https://github.com/Azure/azure-cli/pull/24181

**Comment by scottaddie** [comment]
@xiangyan99 Now that the TSG section exists, are there any exceptions that could be thrown related to this feature? If so, should we link to the relevant TSG section in the exception message?

**Comment by xiangyan99** [comment]
> @xiangyan99 Now that the TSG section exists, are there any exceptions that could be thrown related to this feature? If so, should we link to the relevant TSG section in the exception message?

No. There is no new exception.

**Pull request merged**